README.md

```diff
@@ -3,7 +3,7 @@
 Community translation (French to English) of A. Grothendieck's EGA.
 On est désolé, Grothendieck.
 
-View online [here](https://ega.fppf.site/).
+~~View online [here](https://ega.fppf.site/).~~
```

Comment:
Perhaps you could put the link in the repo's description https://ega.fppf.site here(?)

Reply:
the web-version is currently down, so the repo description links to the (working) pdfs at the moment :-)

Reply:
Is there a Gerby issue I can help with, or did you decide to move away from using Gerby?

Reply:
oh, thanks for offering to help! I'm not actually too sure though — setting up all of the web stuff has been entirely @ryankeleti , and I have no idea how he's done most of it

Reply:
Ah well, it seems it's down again! Probably some weird server issue, or it failed to compile at some point. I'll investigate soon and figure out of it's something up with the deploy or the code.

Reply:
great, thanks :-) and let me know if there's anything i can do to help!

Reply:
https://github.com/ryankeleti/ega/issues/199